### DISCUSSION TITLE
Is there a fast training shortcut for unit testing in mmdetection similar to PyTorch Lightning's Fast Dev Run?

### DISCUSSION THREAD

**Discussion by ryan-minato** [open-mmlab/mmdetection#10313] [Q&A]
Is there a shortcut similar to PyTorch Lightning's Fast Dev Run for mmdetection that allows for a quick round of training for unit testing without saving records and checkpoints? 
If so, could you please provide relevant information and documentation?

like this: https://lightning.ai/docs/pytorch/stable/common/trainer.html#fast-dev-run

**Comment by narthchin** [comment]
You can follow the debug tricks here:
https://github.com/open-mmlab/mmengine/blob/main/docs/en/common_usage/debug_tricks.md
Set `indices` in your dataset config to use a relatively small number of samples for debugging, note that in order to use this feature, your dataset has to be inherited from `BaseDataset`. And for the checkpoints, you can simply remove `CheckpointHook` in your hook config.
